### DISCUSSION TITLE
On search results pages where there are Multiselect Dropdowns should we move active items to the top of the list

### DISCUSSION THREAD

**Discussion by jaymcp** [amnestywebsite/humanity-theme#74] [Enhancement Requests]
In several areas of the site, we have a custom multiselect dropdown element, primarily for selection of taxonomy terms.
On page load, we could re-order the items within these elements such that active items appear at the top of the list.
This would make it easier for the user to see which items they have selected — and to deselect, if required — without having to scroll through the list, particularly if said list is extensive, as in the case of Countries.
Example:
![](http://bigbite.im/i/qnLPlP+)

**Comment by AmnestyAM** [comment]
I like this idea a lot
